code/modules/mapping/random_rooms.dm

```diff
@@ -907,4 +907,12 @@
 	mappath = "_maps/RandomRooms/10x10/sk_rdm106_sanitarium.dmm"
 	centerspawner = FALSE
 	template_height = 10
-	template_width = 10
\ No newline at end of file
+	template_width = 10
+
+/datum/map_template/random_room/sk_rdm106
```

Comment:
10x10s all have deflated weighting. This room will pop up way too often for this niche gimmick- put it around 4-6